src/ducks/SANCharts/ChartPage.js

```diff
@@ -391,7 +391,8 @@ class ChartPage extends Component {
       isLoggedIn,
       isPRO,
       isBeta,
-      alwaysShowingMetrics = []
+      alwaysShowingMetrics = [],
+      isPageLoading
```

Comment:
Why not isLoading? @GarageInc

Reply:
'isLoading' is used here: https://github.com/santiment/app/pull/767/files/38ccfbfd72f5775c90934504b3fcd0008c94c059#diff-28220447578a85d78719e1a7d0dec143R543

But maybe add aliase **_isLoading: isParentLoading_** ?